src/SAML2/XML/saml/AuthnContextDecl.php

```diff
@@ -68,7 +67,7 @@ public function setDecl(DOMElement $decl): void
      */
     public static function fromXML(DOMElement $xml): object
     {
-        return new self($xml);
+        return new self(new Chunk($xml));
```

Comment:
I'm not sure about this one. What you would pass to the `fromXML()` methods are the elements themselves, right? In this case, the `<saml:AuthnContextDecl>` element. The chunk in this case would then be the contents inside `$xml`.

Reply:
Hmm, then how is it possible for the tests to pass?

Reply:
Hmm, I see I do pass the elements themselves..

Reply:
I guess you've discovered a bug.. I've commit a fix